Beta docs (#931)
microsoftgraph/entra-powershell: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Set-EntraBetaApplicationProxyApplicationConnectorGroup.md

```

## Description

The `Set-EntraBetaApplicationProxyApplicationConnectorGroup` cmdlet sets the connector group assigned for the specified application. Specify `ObjectId` and `ConnectorGroupId` parameter to assign the given connector group to a specified application.

The application must be configured for Application Proxy in Microsoft Entra ID.

## Examples

### Example 1: Set a new Connector Group for a specific application

```powershell
Connect-Entra -Scopes 'Directory.ReadWrite.All'
$params = @{
    ObjectId = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb' 
    ConnectorGroupId = 'bbbbbbbb-1111-2222-3333-cccccccccccc'
}
Set-EntraBetaApplicationProxyApplicationConnectorGroup @params
```

This example set a new Connector Group for a specific application.

- `ObjectId` parameter specifies the application ID.
- `ConnectorGroupId` parameter specifies the connector group ID that assign to the application.

## Parameters

### -ConnectorGroupId

The ID of the Connector group that should be assigned to the application.

> You can find the Connector Group ID by using the `Get-EntraBetaApplicationProxyConnectorGroup` command.

Reviewer: ```suggestion
Use the `Get-EntraBetaApplicationProxyConnectorGroup` command to find the Connector Group ID.
```

Author: Changes done.
